index.html

```diff
@@ -229,8 +154,8 @@ <h2 class="text-white">airbnb</h2>
     </section>
 
     <footer class="container-fluid bg-white mb-5 py-5 d-flex justify-content-center gap-5">
-        <img src="/images/logo.svg" alt="" width="200px">
-        <p>2015 Creatove Commons Korea Some Rights Reserved.</p>
+        <img src="images/logo.svg" alt="" width="200px">
```

Comment:
[OPTIONAL] You added an alt attribute but please add a definition for the alt attribute for accessibility and SEO reasons.